### ISSUE TITLE
Design to support inclusion of sub-areas in A&R

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/bcparks-ar-admin#395]
#### Description:

Design and requirements exploration for sub-areas within A&R. This is a first step to ticket #353 .

Note on deadlines:
	- depends on the level of complexity of this ticket and how much work it would be
	- Season launch happens in April, so if we want to incorporate these for next season (again, not a hard deadline), we would need to have it done before then.

#### Acceptance Criteria:
- [ ] Meet with A&R users to understand and write down user needs
- [ ] Create simple wireframes with devs as initial iteration
- [ ] get feedback on designs
- [ ] update implementation ticket (#353) with AC according to designs and user needs

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Standardized variances (includes sub-areas) - working copy - https://bcgov.sharepoint.com/:x:/r/teams/031072/Shared%20Documents/Attendance%20and%20Revenue/Variance/Standardised%20Variances-%20working%20copy.xlsx?d=w6e4eda1261f1465fb8036c7d96ff7fad&csf=1&web=1&e=HMUkq2

**Comment by LindsayMacfarlane** [comment]
Hi @pavelcreates  @meredithom - tagging you directly in this ticket for async backlog refinement. Please review criteria and add anything that is missing. Let me know if you have any questions.

**Comment by pavelcreates** [comment]
From what I can gauge - the acceptance criteria here makes sense. If anything, we can update the ticket once exploration starts.

**Comment by meredithom** [comment]
[Figma Designs](https://www.figma.com/design/yoMhvO0KZvsM6qoqwuIjpt/Attendance-%26-Revenue?node-id=6893-37896)

Home page and menu will also need [updates](https://www.figma.com/design/yoMhvO0KZvsM6qoqwuIjpt/Attendance-%26-Revenue?node-id=5933-36664)

Will add design details to the development ticket once it's been approved.

**Comment by LindsayMacfarlane** [comment]
Feedback received through different channel from @meredithom:

1. The fields were approved in the meeting I had with the A&R team, they're the same ones A&R is using right now to submit a request for a new subarea. I doubled checked with Cam, and it sounds like A&R runs a nightly check with the Data Register. It might be out of scope to auto-populate when adding a new one but it would be nice if possible.

2. It can be left aligned no problem 

Approving designs. Ready for implementation :)

Thank you @meredithom!
